R/convert_se_to_raw_data.R

```diff
@@ -23,13 +23,11 @@ convert_mae_to_raw_data <- function(mae) {
   # Remove duplicates shared between assays to keep only original single-agent
   common_records <- Reduce(intersect, lapply(data, "[[", "record_id"))
   sa_name <- gDRutils::get_experiment_groups("single-agent")[["single-agent"]]
-  data[setdiff(names(data), sa_name)] <- 
-    lapply(data[setdiff(names(data), sa_name)], function(x) {
-      x[!record_id %in% common_records]
-    })
-  
+  if (length(names(data) > 1)) {
```

Comment:
Please, add tests for scenario `length(names(data) > 1`

Reply:
Added in [2317075](https://github.com/gdrplatform/gDRcore/pull/145/commits/23170753f42755e1b865c6b0e1dcfd084f71b8bb)